lib/DDGC/Web/Controller/InstantAnswer.pm

```diff
@@ -17,10 +18,20 @@ use if debug, 'Data::Dumper';
 
 sub base :Chained('/base') :PathPart('ia') :CaptureArgs(0) {
     my ( $self, $c ) = @_;
+
+    my $last_modified = $c->d->rs('InstantAnswer')->last_modified;
+    $last_modified->formatter( 'DateTime::Format::HTTP' );
+
+    $c->response->header (
+        'Last-Modified' => $last_modified;
+    );
```

Comment:
@jbarrett it depends on the endpoint, pretty much. We have a bunch that fetch all the IAs at once.
Or were you asking whether we always use the TO_JSON function for fetching the data?